test/tsconfig.json

```diff
@@ -142,4 +143,4 @@
         "rule-tester/testData.ts",
         "rule-tester/utilsTests.ts"
     ]
-}
+}
```

Comment:
Can you re-add the newline at the end of this file? (So we prevent merge conflicts and unnecessary diffs)